### ISSUE TITLE
2023BPix Jet Veto Map Issue

### ISSUE THREAD

**Issue by MatthewDittrich** [cmstas/ewkcoffea#40]
![Screen Shot 2024-07-27 at 9 24 24 PM](https://github.com/user-attachments/assets/bdadf5af-af40-494a-a206-43adf9a8f16e)

**Comment by kmohrman** [comment]
@MatthewDittrich is this addressed in the veto map implementation in PR #53?

**Comment by MatthewDittrich** [comment]
this is not addressed. There is some conflicting information in the correctionlib file. I will look into it and we can chnage later if need be

**Comment by kmohrman** [comment]
@MatthewDittrich: Just for future reference, wondering if you could comment on what ended up happening here and what the final state of this was for the WWZ analysis?

**Comment by MatthewDittrich** [comment]
As of June 30 2025, the current recommendation is to use the nominal `jetvetomap` as you can see [here](https://cms-jerc.web.cern.ch/Recommendations/#jet-veto-maps). The jetvetomap for 2023BPix likely needs to be updated for any future work as shown in this discussion (https://cms-talk.web.cern.ch/t/updated-jetvetomap-for-summer23bpixprompt23/97635)

For SMP-24-015, we vetoed the BPix region (which was the default recommendation), but this can be handled better with a BPix map. However, once we update the the newer map, this is done by default.
